assets/js/app.js

```diff
@@ -56,7 +56,17 @@ Hooks.CommentPreview = {
   },
 };
 let csrfToken = document.querySelector("meta[name='csrf-token']").getAttribute('content');
-let liveSocket = new LiveSocket('/live', Socket, {hooks: Hooks, params: {_csrf_token: csrfToken}});
+let liveSocket = new LiveSocket('/live', Socket, {
+  hooks: Hooks,
+  params: { _csrf_token: csrfToken },
+  metadata: {
+    keydown: (event, _element) => ({
+      key: event.key,
+      metaKey: event.metaKey,
+      ctrlKey: event.ctrlKey,
+    }),
+  },
```

Comment:
In order for info about modifier keys to be sent to the backend, we need to add it manually. Docs [here](https://hexdocs.pm/phoenix_live_view/bindings.html#key-events).